pyaerocom/griddeddata.py

```diff
@@ -2532,10 +2532,18 @@ def std(self):
 
     def _check_lonlat_bounds(self):
         """Check if longitude and latitude bounds are set and if not, guess"""
-        if self.longitude.bounds is None:
-            self.longitude.guess_bounds()
-        if self.latitude.bounds is None:
-            self.latitude.guess_bounds()
+        if not self.longitude.has_bounds():
```

Comment:
I do not think the test added really covers this section.
Here the exception is silenced, but the test relies on the exception beer raised.

Reply:
yeah, the ValueError exception is actually raised in iris. 
```
python3.11/site-packages/iris/coords.py:2384: in guess_bounds
```
But I needed to the pyaerocom code to pass through that exception to make the analysis work. So it's not really testable.